course/views.py

```diff
@@ -392,17 +392,28 @@ def _get_problem_result(solution, request):
             tango_time = timezone.make_aware(
                 tango_time, timezone=timezone.UTC())
             if tango_time != prob_result.timestamp:
-                if "Autodriver: Job timed out after " in line:  # thats the text that Tango outputs when a job times out
-                    prob_result.score = 0
-                    prob_result.json_log = {'score_sum': '0', 'external_log': [
-                        "Program timed out after " + line.split(" ")[-2] + " seconds."]}
-                    prob_result.timestamp = tango_time
-                    prob_result.raw_output = raw_output
-                    prob_result.save()
-                else:
+                timeout = False
+                elided = False
+                for line in r.text.split("\n"):
+                    if "Autodriver: Job timed out after " in line:  # thats the text that Tango outputs when a job times out
+                        prob_result.json_log = {'score_sum': '0', 'external_log': [
+                            "Program timed out after " + line.split(" ")[-2] + " seconds."]}
+                        prob_result.timestamp = tango_time
+                        timeout = True
+                        break
+                    elif "...[excess bytes elided]..." in line:
```

Comment:
When I run it on my local machine it says [excess bytes elided] somewhere in the middle of the file. I'm going to run it on the actual server and see what happens.